React/Views/RCTView.m

```diff
@@ -16,6 +16,9 @@
 #import "RCTUtils.h"
 #import "UIView+React.h"
 #import "RCTI18nUtil.h"
+#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
```

Comment:
Can we reference an open source Issue instead?

Reply:
There isn't a equiv github issue: this is the conventional comment for any code changes that are a macOS delta from upstream.